CHANGES.md

```diff
@@ -47,8 +47,10 @@ To be released.
     given.
  -  Fixed a bug that TURN relay had been disconnected when being connected for longer than 5
     minutes.
+ -  Improved connection stability of TURN relay.
  -  Instead of validating the entire blocks, `BlockChain<T>.Append()` method
     became to validate only the next block to be appended.  [[#210]]
+ -  Improved `BlockChain<T>.Fork()` performance by omitting unnecessary validation.
```

Comment:
```suggestion
 -  Improved `BlockChain<T>.Fork()` performance by avoiding double validation
    of already validated blocks.  [[#215]]
```

Reply:
Also need to add `[#215]: https://github.com/planetarium/libplanet/pull/185` below.